src/scss/styles.scss

```diff
@@ -1,2 +1,3 @@
 @use 'uswds-core';
 @forward 'uswds';
+@use 'image_example';
```

Comment:
> This would ensure it's included in the precompiled CSS version of the package, but it would not be included the [Sass entrypoint](https://github.com/18F/identity-design-system#styles), which should be expected to include all components of the design system.

After some research, I'm still not sure I'm fully understanding the difference here, but I've made that change ([commit](https://github.com/18F/identity-design-system/pull/369/commits/2c424274b745bf2c6280f488bce5b0c8fb06d139)).

I also prepended `usa-` ([commit](https://github.com/18F/identity-design-system/pull/369/commits/51100fca9ed723957325dcc1e7bb3947b93818ea)).

Reply:
Thanks for spelling that out, this makes a lot more sense to me now.